tests/unit/helpers/test_cli.py

```diff
@@ -79,7 +80,30 @@ def test_create_artifact_payload_from_filepath(
         }
 
         assert (
-            create_artifact_payload_from_filepath(
+            create_artifact_add_payload_from_filepath(
+                filepath=temp_file,
+                path=path,
+            )
+            == expected_artifact_payload
+        )
+
+    def test_create_artifact_delete_payload_from_filepath(
```

Comment:
One more test will be useful:
`create_artifact_add_payload_from_filepath_without_path`